app/src/main/java/com/example/caloriecounter/feature/restaurant/ui/meals/MealList.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.caloriecounter.feature.restaurant.ui.meals
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun MealList(category: String) {
+    Text(
+        text = "$category MealList",
+        fontSize = 24.sp,
```

Comment:
nit: better to avoid hardcoded values for Typography `fontSize`

Reply:
Yes, this `Text` will be deleted after creating real `MealListComponent`.

Reply:
Oh, I see 👍🏻